src/test/java/io/confluent/idesidecar/restapi/messageviewer/RecordDeserializerTest.java

```diff
@@ -352,7 +363,7 @@ record TestCase(
     ) {
       static TestCase nonRetryable(int statusCode) {
         // Expect only 1 try when the status code is non-retryable
-        return new TestCase(statusCode, 3, 1, null);
+        return new TestCase(statusCode, 1, 1, null);
```

Comment:
Hmm, did we fix a bug in the implementation (and hence in the test)?

Reply:
This change should not have any affect because `assertSchemaFetchFailuresCached(...)`'s parameter `maxRetries` seems to be unused. We might want to refactor this in a follow-up PR.